spec/ext/marc/record_spec.rb

```diff
@@ -153,63 +159,43 @@ def stub_builder(_001, _003, _035s)
 
   describe 'm300_without_a' do
     
-        rec1 = MARC::Record.new
-        rec1 << MARC::DataField.new('300', ' ', ' ', ['a', ''])
-        rec1 << MARC::DataField.new('300', ' ', ' ', ['b', ''])
-        it 'is true if any 300 lacks 300$a' do
-          expect(rec1.m300_without_a?).to be true
-        end
-    
-        rec2 = MARC::Record.new
-        rec2 << MARC::DataField.new('300', ' ', ' ', ['a', ''], ['b', ''])
-        rec2 << MARC::DataField.new('300', ' ', ' ', ['z', ''], ['a', ''])
-        it 'is nil if all 300s have 300$a' do
-          expect(rec2.m300_without_a?).to be_nil
-        end
-
-        rec4 = MARC::Record.new
-        it 'is nil if no 300s exist' do
-          expect(rec4.m300_without_a?).to be_nil
-        end
-    
-      end
-
-
-  describe 'no_oclc_035' do
-
-    it 'is true if 035(s) are non-OCLC' do
-      r = stub_builder('123', 'ItFiC', ['(Nonsense)567'])
-      expect(r.no_oclc_035?).to be true
+    rec1 = MARC::Record.new

```

Comment:
I think the diff gets confused starting here. This test stayed the same. Added a test for counting fields for a given tag in a record. removed tests for none/multiple OCLC 035s (we're now just counting the results of get_035oclcnums)